addons/core/translations/resources/en-us.yaml

```diff
@@ -827,15 +827,25 @@ credential-store:
   messages:
     welcome:
       title: Welcome to Credential Stores
-    no-filter:
-      title: No worker filter added
-      description: You haven't added a worker filter yet
   titles:
     new: New Credential Store
-    worker-filter: Worker Filter
+  worker-filter:
+    messages:
+      no-filter:
+        title: No worker filter added
+        description: You haven't added a worker filter yet
+      formatted-worker-filter:
+        description: The worker filter format for this credential store
+    titles:
+      worker-filter:
+        title: Worker Filter
+      edit-worker-filter:
+        title: Edit Worker Filter
+        description: Specify workers that have access to the credential store
   actions:
-    delete: Delete Credential Store
     add: Add worker filter
```

Comment:
small nitpick: can the add translation be updated to be `Add Worker Filter` to match the other actions

Reply:
Yep, but that text lives inside the template whereas the delete is in the dropdown. Not sure if that changes your preference.

Lmk!

Current:
<img width="304" alt="Screenshot 2024-11-21 at 4 31 14 PM" src="https://github.com/user-attachments/assets/eee53285-a58f-4a10-b5b9-52e586a30fef">

With update:
<img width="304" alt="Screenshot 2024-11-21 at 4 32 41 PM" src="https://github.com/user-attachments/assets/53a71943-0639-40d4-b52b-5809f495e795">

Reply:
Im thinking thats fine since we do that other places where we have an app state:
<img width="335" alt="Screenshot 2024-11-21 at 1 41 35 PM" src="https://github.com/user-attachments/assets/2cc81ca1-582e-4b52-8139-18a3502d060b">